playback/src/mixer/softmixer.rs

```diff
@@ -43,7 +43,7 @@ impl Mixer for SoftMixer {
 }
 
 impl SoftMixer {
-    pub const NAME: &'static str = "softmixer";
+    pub const NAME: &'static str = "softvol";
```

Comment:
This is what is has been called up to and including 0.2.0 so let's not change now.